### ISSUE TITLE
Is multi-card joint training necessary? Can a single-card V100 do such an experiment?

### ISSUE THREAD

**Issue by yrfmessi** [GeorgeLuImmortal/PUnifiedNER#3]

**Comment by GeorgeLuImmortal** [comment]
I think you can, you can reduce the batch size and use gradient accumulation to simulate a large batch training. If still OOM you can try to use fewer sequent lengths
